.travis.yml

```diff
@@ -156,7 +156,7 @@ matrix:
         - find "$NODE_DIR" -type d -empty -delete
         - nvm install
       install:
-        - yarn install --pure-lockfile
+        - ./bin/yarn install --pure-lockfile
```

Comment:
Are these changes supposed to be in particular PR?

Reply:
Looks like a funky rebase.